src/main/java/seedu/address/model/person/Email.java

```diff
@@ -9,23 +9,17 @@
  */
 public class Email {
 
-    private static final String SPECIAL_CHARACTERS = "!#$%&'*+/=?`{|}~^.-";
-    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@domain "
+    private static final String SPECIAL_CHARACTERS = ".!#$%&'*+/=?`{|}~^.-";
+    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@label(.label)*"
```

Comment:
Simplify to English for the format portion, nobody will understand the brackets and asterisk. `Emails should be of the format "local-part@label". Multiple labels can follow after provided that they are prefixed with a single period.`

Reply:
I feel that this expression is adequate. If the user doesn't get it he can just refer to point 3. I'll add an example in the usage message

Reply:
Hmm, up to you. But I still opt for longer explanations for better clarity than brevity.